script/release.sh

```diff
@@ -0,0 +1,292 @@
+#!/bin/bash
+
+# Constants. Enter relevant repo information here.
+UPSTREAM_REPO="kubernetes-incubator"
+CLI="kompose"
+
+usage() {
+  echo "This will prepare $CLI for release!"
+  echo ""
+  echo "Requirements:"
+  echo " git"
+  echo " gpg - with a valid GPG key already generated"
+  echo " tar - you should already have this on your system"
+  echo " hub"
+  echo " github-release"
+  echo " GITHUB_TOKEN in your env variable"
+  echo " "
+  echo "Not only that, but you must have permission for:"
+  echo " Tagging releases within Github"
+  echo ""
+}
+
+requirements() {
+  if [ ! -f /usr/bin/git ] && [ ! -f /usr/local/bin/git ]; then
+    echo "No git. What's wrong with you?"
+    return 1
+  fi
+
+  if [ ! -f /usr/bin/gpg ] && [ ! -f /usr/local/bin/gpg ]; then
+    echo "No gpg. What's wrong with you?"
+    return 1
+  fi
+
+  if [ ! -f $GOPATH/bin/github-release ]; then
+    echo "No $GOPATH/bin/github-release. Please run 'go get -v github.com/aktau/github-release'"
+    return 1
+  fi
+
+  if [ ! -f /usr/bin/hub ]; then
+    echo "Hub needed in order to create the relevant PR's. Please install hub @ https://github.com/github/hub"
+    return 1
+  fi
+
+  if [[ -z "$GITHUB_TOKEN" ]]; then
+    echo "export GITHUB_TOKEN=yourtoken needed for using github-release"
+  fi
+}
+
+# Clone and then change to user's upstream repo for pushing to master / opening PR's :)
+clone() {
+  git clone ssh://git@github.com/$UPSTREAM_REPO/$CLI.git
```

Comment:
@sebgoa yes, git is used a lot in the script, much easier over ssh authentication.